modules/standards/package.json

```diff
@@ -4,42 +4,29 @@
   "description": "",
   "repository": "phenyl-js/phenyl",
   "license": "Apache License 2.0",
-  "author": "Shin Suzuki <shinout310@gmail.com> ,Nicholas Lee <nicholaslee119@gmail.com>",
+  "author": "Shin Suzuki <shinout310@gmail.com>",
   "files": [
-    "src",
-    "dist",
-    "package.json",
-    "jsnext.js",
+    "lib",
     "LICENSE",
-    "README.md",
-    "decls"
+    "README.md"
   ],
-  "main": "lib/index.js",
-  "types": "lib/index.d.js",
+  "main": "lib/index.ts",
+  "types": "lib/index.d.ts",
   "scripts": {
     "build": "upbin tsc --declaration",
     "lint": "upbin eslint 'src/**/*.ts' 'test/**/*.ts' --fix",
     "test": "upbin mocha --require ts-node/register 'test/**/*.ts' --color always",
     "type-check": "upbin tsc --noEmit"
   },
   "dependencies": {
-    "@phenyl/interfaces": "*",
-    "@phenyl/utils": "*",
-    "@sp2/format": "~1.2.3",
-    "@sp2/updater": "~1.1.0",
-    "mongolike-operations": "~0.1.5",
-    "oad-utils": "~0.3.0",
-    "phenyl-interfaces": "~0.4.2",
-    "phenyl-utils": "~0.4.2",
+    "@phenyl/interfaces": "~1.0.0",
+    "@phenyl/utils": "~1.0.0",
+    "sp2": "~1.4.1",
     "power-assign": "~0.2.10",
     "power-crypt": "~0.1.4"
   },
   "devDependencies": {
-    "@types/mocha": "^5.0.5",
-    "@babel/register": "^7.0.0",
-    "mocha": "^5.0.5",
-    "nyc": "^11.6.0",
-    "power-assert": "^1.4.4",
     "upbin": "^0.8.1"
-  }
+  },
+  "collaborators": "Nicholas Lee <nicholaslee119@gmail.com>"
```

Comment:
thie key should be `contributors` instead of `collaborators` [reference link](https://docs.npmjs.com/files/package.json#people-fields-author-contributors)